### ISSUE TITLE
v3.2.1 build failure on arm64 macOS

### ISSUE THREAD

**Issue by p-linnane** [simdutf/simdutf#218]
Hello 👋 . I'm a maintainer for the [Homebrew](https://brew.sh) project. While packaging the latest release of simdutf we are encountering build failures on arm64 versions of macOS. The failures are as follows:
```
  duplicate symbol 'setup_performance_counters()' in:
      CMakeFiles/benchmark.dir/benchmark.cpp.o
      src/libsimdutf_benchmarks_benchmark.a(benchmark_base.cpp.o)
  duplicate symbol '_regs' in:
      CMakeFiles/benchmark.dir/benchmark.cpp.o
      src/libsimdutf_benchmarks_benchmark.a(benchmark_base.cpp.o)
  duplicate symbol '_counter_map' in:
      CMakeFiles/benchmark.dir/benchmark.cpp.o
      src/libsimdutf_benchmarks_benchmark.a(benchmark_base.cpp.o)
  duplicate symbol '_counters_1' in:
      CMakeFiles/benchmark.dir/benchmark.cpp.o
      src/libsimdutf_benchmarks_benchmark.a(benchmark_base.cpp.o)
  duplicate symbol '_counters_0' in:
      CMakeFiles/benchmark.dir/benchmark.cpp.o
      src/libsimdutf_benchmarks_benchmark.a(benchmark_base.cpp.o)
  duplicate symbol 'setup_performance_counters()' in:
      CMakeFiles/benchmark.dir/benchmark.cpp.o
      src/libsimdutf_benchmarks_benchmark.a(benchmark.cpp.o)
  duplicate symbol '_regs' in:
      CMakeFiles/benchmark.dir/benchmark.cpp.o
      src/libsimdutf_benchmarks_benchmark.a(benchmark.cpp.o)
  duplicate symbol '_counter_map' in:
      CMakeFiles/benchmark.dir/benchmark.cpp.o
      src/libsimdutf_benchmarks_benchmark.a(benchmark.cpp.o)
  duplicate symbol '_counters_1' in:
      CMakeFiles/benchmark.dir/benchmark.cpp.o
      src/libsimdutf_benchmarks_benchmark.a(benchmark.cpp.o)
  duplicate symbol '_counters_0' in:
      CMakeFiles/benchmark.dir/benchmark.cpp.o
      src/libsimdutf_benchmarks_benchmark.a(benchmark.cpp.o)
  ld: 10 duplicate symbols for architecture arm64
```

Relates to https://github.com/Homebrew/homebrew-core/pull/124086.

**Comment by lemire** [comment]
Please try version 3.2.2.

**Comment by p-linnane** [comment]
Fixed in 3.2.2. Thank you!
